pkgs/development/libraries/mtxclient/default.nix

```diff
@@ -1,5 +1,17 @@
-{ stdenv, fetchFromGitHub, fetchpatch, cmake, pkgconfig
-, boost, openssl, zlib, libsodium, olm, nlohmann_json }:
+{ stdenv
+, fetchFromGitHub
+, fetchpatch
+, cmake
+, pkgconfig
+, boost
+, openssl
+, zlib
+, libsodium
+, olm
+, spdlog
+, gtest
```

Comment:
Commit message says "mtxclient: format inputs and arguments", but you add unused `gtest` argument.